src/pages/index.tsx

```diff
@@ -17,6 +18,22 @@ export default function Page() {
     trackPageVisit();
   }, []);
 
+  const handleScroll = debounce((e) => {
+    const bodyScroll = e.target.documentElement.scrollTop;
+    if (bodyScroll > 50) {
+      document.body.classList.add('scrolled');
+    } else if (document.body.classList.contains('scrolled')) {
+      document.body.classList.remove('scrolled');
+    }
+  });
+
+  React.useLayoutEffect(() => {
```

Comment:
What's the reason for using `React.useLayoutEffect` instead of importing it?

Reply:
I think you typically want to add event listeners inside of useEffect or useLayoutEffect so that it can also be removed and you don't end up with weird side effects of the event handler persisting.

And (I think, anyways) that useLayoutEffect is preferred for events like scroll events, measuring UI elements via getBoundingClientRect(), etc.

Reply:
Ah sorry I wasn't clear. I just noticed that you had the `React.` before the `useLayoutEffect`. Looking at the imports it looks like we're already importing the `useEffect` and so I was wondering if there was a reason for accessing `useLayoutEffect` from within the `React` import.

Reply:
oh! lollll my bad.  Good callout, I tidied those up in the latest.